### PR TITLE
AWS::CloudFormation::Interface

### PR DESCRIPTION
**Pull request by dylanvaughn**
@acarlton @helen-kim-oe please take a look

### CODE REVIEW

File: cdk/drupal/drupal_stack.py [hunk 1]
```diff
@@ -829,31 +840,30 @@ def __init__(self, scope: core.Construct, id: str, **kwargs) -> None:
...
-            description="The number of cache nodes in the memcached cluster.",
+            description="Required: The number of cache nodes in the memcached cluster (only applies ElastiCache enabled).",
             min_value=1,
             max_value=20,
             type="Number"
         )
         elasticache_enable_param = core.CfnParameter(
             self,
-            "ElastiCacheEnable",
+            "_ElastiCacheEnable",
```
Comment (acarlton) [inline cdk/drupal/drupal_stack.py:863]: Is there a reason this name in particular is prefaced with _ ?

File: cdk/drupal/drupal_stack.py [hunk 2]
```diff
@@ -966,7 +976,7 @@ def __init__(self, scope: core.Construct, id: str, **kwargs) -> None:
             "CloudFrontOriginAccessPolicyParam",
             allowed_values = [ "http-only", "https-only", "match-viewer" ],
             default="match-viewer",
```
Comment (acarlton) [inline cdk/drupal/drupal_stack.py:978]: @dylanvaughn I think we have some issues with the Cloudfront-Origin configuration that aren't quite smoothed out. I debugged some 502 gateway issues last week and now I think there is more to look into, but pending that investigation, do you think this parameter could get us into trouble? If the customer doesn't supply a CertificateARN, CloudFront would be unable to access the origin via HTTPS, so the https-only option wouldn't work...

File: cdk/drupal/drupal_stack.py [hunk 3]
```diff
@@ -393,9 +405,9 @@ def __init__(self, scope: core.Construct, id: str, **kwargs) -> None:
             "Properties.VpcId",
             {
                 "Fn::If": [
-                    customer_vpc_not_given_condition.logical_id,
+                    vpc_not_given_condition.logical_id,
                     vpc.ref,
-                    customer_vpc_id_param.value.to_string()
+                    vpc_id_param.value.to_string()
```
Comment (acarlton) [inline cdk/drupal/drupal_stack.py:410]: In other places in the code, we are using the parameter.value_as_string attribute.

File: cdk/drupal/drupal_stack.py (hunk 1 above)
Comment (dylanvaughn) [reply cdk/drupal/drupal_stack.py:863]: Good catch - I was playing with logical ids to try to get the ordering right but turns out it wasn't necessary.

File: cdk/drupal/drupal_stack.py (hunk 2 above)
Comment (dylanvaughn) [reply cdk/drupal/drupal_stack.py:978]: @acarlton I created a ticket to look into this more.

File: cdk/drupal/drupal_stack.py (hunk 3 above)
Comment (dylanvaughn) [reply cdk/drupal/drupal_stack.py]: I updated this most places but we still need to use `value.to_string()` for `Number` param types.

**Pull request merged**